### PR TITLE
feat(es6-class): add support for es6 classes

### PR DESCRIPTION
**Pull request by tpai**
Add es6 class support and test code.

### CODE REVIEW

File: test/module.spec.js [hunk 1]
```diff
@@ -78,6 +78,24 @@ describe('debuk', () => {
...
+          return `${this.name}: ${words}`;
+        }
+      }
+      const PerfClass = debuk(MyClass, { params: true });
+      const args = ['Tom', 'Mary'];
+
+      args.forEach(input => {
+        const results = new MyClass(input).speak('test');
+        const perf = new PerfClass(input).speak('test');
+        expect(perf).to.be.equal(results);
```
Comment (udnisap) [inline test/module.spec.js]: shall we assert on the params as well?

File: test/module.spec.js [hunk 2]
```diff
@@ -78,6 +78,24 @@ describe('debuk', () => {
         expect(perfFn(...input)).to.be.equal(results);
       });
     });
+    it('Should return the called class when executed', () => {
```
Comment (udnisap) [inline test/module.spec.js]: new tests to check for multiple method class with debuk called

**Comment by udnisap** [comment]
@tpai can you also update the README.md?

**Comment by tpai** [comment]
@udnisap done

**Pull request merged**